book/src/getting-started/1-qobjects-in-rust.md

```diff
@@ -0,0 +1,77 @@
+<!--
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
+-->
+
+# QObjects in Rust
+
+> The right tool for the right job.
+
+> If you only have a hammer, every problem looks like a nail.
+
+> Don't bring a knife to a gun fight.
+
+There are many bits of advice like that.
+With CXX-Qt, we aim to make it possible to use the right tool for each of the many jobs necessary to build a modern GUI application.
+
+So what is in our toolbox for a typical Qt application?
+- QML - A declarative, flexible, dynamically-typed, interpreted language that is purpose built to define reactive and beautiful GUI layouts and widgets with quick iteration speed.
+- C++ - The traditional back-end of Qt - A fast, low-level language with a strong type system. C++ offers a rich ecosystem, many Qt-specific libraries and bare-metal performance. The cost when using C++ is that it is slow to develop, very error-prone and can easily lead to memory-issues, which can instantly crash your application and cause security issues.
+
+Notably absent then is a back-end language that allows us to get rid of the issues C++ has and provides us with a safe way to write fast Back-end code.
```

Comment:
back-end vs Back-end in the same sentence.

Reply:
Ah, stupid german brain capitalizing all the Nouns ;)